src/azul/service/manifest_controller.py

```diff
@@ -137,28 +140,47 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            instructions = dedent(f'''
+            {_curl_instructions(os_system='Unix', cmd=f'curl "{location}"')}
+            {_curl_instructions(os_system='Windows', cmd=f'curl.exe "{location}"')}
+            ''')
             body = {
                 'Status': 301,
                 'Location': location.url,
                 'Retry-After': token.wait_time
             }
         else:
-            if fetch:
+            try:
+                query_params['objectKey']
+            except KeyError:
                 url = self.manifest_url_func(fetch=False,
                                              catalog=manifest.catalog,
                                              format_=manifest.format_,
                                              filters=json.dumps(manifest.filters),
                                              objectKey=manifest.object_key)
             else:
                 url = manifest.location
+            command_lines = self.service.command_lines(manifest, url)
+            instructions = dedent(f'''
+            {_curl_instructions(os_system='Unix', cmd=command_lines['bash'])}
+            {_curl_instructions(os_system='Windows', cmd=command_lines['cmd.exe'])}
+            ''')
+            command_lines = {'CommandLine': command_lines}
             body = {
                 'Status': 302,
                 'Location': url,
-                'CommandLine': self.service.command_lines(manifest, url)
+                **command_lines
```

Comment:
Why not just
```suggestion
                'CommandLine': command_lines
```
without the variable re-assignment?

Reply:
I missed that, `command_lines` is no longer contingent on the manifest format.